Gemfile

```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-git_source(:github) {|repo_name| "https://github.com/melvrickgoh/event_tracer" }
+git_source(:ascenda_private) { |repo_name| ENV['BUNDLE_GITHUB__HTTPS'] ? "https://github.com/#{repo_name}.git" : "git@github.com:#{repo_name}.git" }
```

Comment:
This is a public gem, so we shouldn't use `ascenda_private`

Reply:
I thought we have moved this gem to our private repo. But anw this is a mistake, I forgot to revert it back 🙇